src/applications/disability-benefits/996/utils/helpers.js

```diff
@@ -134,86 +134,6 @@ export const mayHaveLegacyAppeals = ({
   additionalIssues,
 } = {}) => legacyCount > 0 || additionalIssues?.length > 0;
 
```

Comment:
I moved things around in this file to make it more closely match the structure of the same NOD (10182) file.